src/utils/tickets.js

```diff
@@ -30,28 +39,26 @@ const getMalwareMatches = async URLs => {
     },
   };
 
-  return fetch(safeBrowsingURL, {
-    method: 'POST',
-    body: JSON.stringify(request),
-    responseType: 'application/json',
-  })
-    .then(res => {
-      return res.json();
-    })
-    .then(json => {
-      if (json.matches === undefined) {
-        console.log('No malware found');
-        return [];
-      } else {
-        console.log('Malware found');
-        console.log(json.matches);
-        return json.matches;
-      }
-    })
-    .catch(error => {
-      console.log(error);
-      return [];
+  try {
+    // Parse request
+    const res = await fetch(safeBrowsingURL, {
+      method: 'POST',
+      body: JSON.stringify(request),
+      responseType: 'application/json',
     });
+    const json = res.json();
+    if (!json.matches) {
+      console.log('No Malware found');
+      return [];
+    } else {
+      console.log('Malware found');
+      console.log(json.matches);
+      return json.matches;
+    }
+  } catch (err) {
+    console.error(error);
+    throw Error(err.message);
```

Comment:
In the long term, we probably want to catch errors during HC processing, and change states (right now there’s no way to retry, or have a processed ticket for which some subtasks errored).

Reply:
Yeah we definitely have to do some error handling refactoring across the board when we do that big refactor